### PR TITLE
MAYH-6056 Can generate index from command line

### PR DESCRIPTION
**Pull request by newhoggy**

### CODE REVIEW

File: src/App/Index.hs [hunk 1]
```diff
@@ -70,6 +80,47 @@ indexBp opt = do
...
+    .|  compressWordAsBit
+    .|  sinkFileBS (filename <> ".ib")
+
+  runConduitRes
+    $   sourceFileBS (filename <> ".blank")
+    .|  blankedXmlToBalancedParens2
+    .|  compressWordAsBit
+    .|  sinkFileBS (filename <> ".bp")
+
+  ibData <- loadVector (filename <> ".ib")
```
Comment (newhoggy) [inline src/App/Index.hs]: They're just intermediate files that's useful for debugging.

Comment (newhoggy) [inline src/App/Index.hs]: This is only used on command line for local testing.

Comment (vashu1) [inline src/App/Index.hs]: debugging of hw-xml? maybe it should go there?

Comment (vashu1) [inline src/App/Index.hs]: if you think it belongs here \+1

**Pull request merged**